### PR TITLE
BUG: extreme long inline sequences cause tuneTailOpt runaway Tm

### PR DESCRIPTION
**Pull request by daz10000**
Have only seen this once in a hard to document test case but long inline sequences and challenging primer constraints cause tuneTailOpt to make extremely short amplification regions, which then approach 0 Kelvin melting temperatures eventually blowing up Tm cals on a zero length overlap.  This fix constraints the solution space so at least the Tm calcs won't blow up and crash.   Patch also adds some verbose mode only diagnostics to help next poor person dealing with this code.

### CODE REVIEW

File: src/GslCore/PrimerCreation.fs
Comment (chrismacklin) [inline src/GslCore/PrimerCreation.fs]: Please normalize indentation to four spaces per level.

File: src/GslCore/PrimerCreation.fs
Comment (chrismacklin) [inline src/GslCore/PrimerCreation.fs]: It would be nice to extract this threshold as a named constant.

File: src/GslCore/PrimerCreation.fs
Comment (chrismacklin) [inline src/GslCore/PrimerCreation.fs]: Please normalize indentation.

**Review by chrismacklin — CHANGES_REQUESTED** [review]
Looks good aside from few small tweaks.

I'll need to run regression tests before merging.

File: src/GslCore/PrimerCreation.fs
Comment (daz10000) [reply src/GslCore/PrimerCreation.fs]: Done.

**Comment by daz10000** [comment]
Made those changes.  Thanks for attention to detail.   Let me know if regressions give you any trouble.

File: src/GslCore/PrimerCreation.fs
Comment (chrismacklin) [reply src/GslCore/PrimerCreation.fs]: This section is still 8 spaces per indent.

**Comment by chrismacklin** [comment]
Tests are green, I'll merge after last indentation fix.

File: src/GslCore/PrimerCreation.fs
Comment (daz10000) [reply src/GslCore/PrimerCreation.fs]: apologies,  184G7<<

File: src/GslCore/PrimerCreation.fs
Comment (chrismacklin) [inline src/GslCore/PrimerCreation.fs]: These blocks are still 8 spaces :)

**Comment by daz10000** [comment]
This time fur shure rocky - I swear

**Comment by chrismacklin** [comment]
💯

**Review by chrismacklin — APPROVED** [review]

**Pull request merged**
